doc/classes/MultiMeshInstance.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MultiMeshInstance" inherits="GeometryInstance" category="Core" version="3.0-rc1">
 	<brief_description>
-		Node that instances a [MultiMesh].
+		Node that Instances a [MultiMesh].
```

Comment:
Should not be capitalized, since it is not a proper noun or something of that sort. Google "capitalization rules" to find some rules on when words should be capitalized.

Reply:
I usually set Grammarly Plugin and Ginger to Indian English.